tracer/src/Datadog.Trace/Configuration/ImmutableExporterSettings.cs

```diff
@@ -29,11 +29,13 @@ public ImmutableExporterSettings(IConfigurationSource source)
         /// <param name="settings">The tracer settings to use to populate the immutable tracer settings</param>
         public ImmutableExporterSettings(ExporterSettings settings)
         {
-            TraceTransport = settings.TracesTransport;
             AgentUri = settings.AgentUri;
+            TraceTransport = settings.TracesTransport;
```

Comment:
I think this should use the internal type and you can remove the public TraceTransport
```suggestion
            TracesTransport = settings.TracesTransport;
```